src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```diff
@@ -149,3 +149,10 @@ Code snippet
 .cds--snippet {
   width: fit-content;
 }
+
+/******************************************
+Tabs
+******************************************/
+.cds--tab-content {
+  background: var(--cds-layer-01);
```

Comment:
why this ?

Reply:
Hey @afreen23, the angular contained tabs implementation does not fill the background color to match the active tab color. So this takes care of it, by coloring the contined space of the tab.

If you take a look at [React's implementation of this feature](https://react.carbondesignsystem.com/?path=/story/components-tabs--contained): you can see the gray contained space.

Does that no happen in the [Angular's implementation of this feature](https://angular.carbondesignsystem.com/?path=/story/components-tabs--basic&args=type:contained): hence this adds this gray coloring

I though it looked better this way, let me know your thoughts